### PR TITLE
chore: use biome and vitest, remove everything else

### PR DESCRIPTION
**Pull request by aldy505**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+import { type SerializeOptions, serialize } from '@tinyhttp/cookie'
 import { sign } from '@tinyhttp/cookie-signature'
 import { Tokens } from './token'
 
 export interface CSRFRequest extends IncomingMessage {
   csrfToken(): string
   secret?: string | string[]
-  signedCookies?: any
-  cookies?: any
+  signedCookies?: unknown
```
Comment (talentlessguy) [inline src/index.ts:10]: shouldn't these be a Record type?

Comment (aldy505) [reply src/index.ts:10]: I put an `assertRecord()` function below to make sure it's `Record` type

Comment (talentlessguy) [reply src/index.ts:10]: yeah but why not type them already as records?

Comment (aldy505) [reply src/index.ts:10]: I forgot why I didn't do that.

Comment (talentlessguy) [reply src/index.ts:10]: could you update it then? i don't think we need a runtime assertion

Comment (aldy505) [reply src/index.ts:10]: Sure will do that tomorrow

**Comment by aldy505** [comment]
ping @talentlessguy

**Comment by talentlessguy** [comment]
there's a coverage decrease idk why but I'll merge it

**Pull request merged**